rllib/agents/sac/sac_policy.py

```diff
@@ -95,37 +95,26 @@ def get_dist_class(config, action_space):
     if isinstance(action_space, Discrete):
         action_dist_class = Categorical
     else:
-        action_dist_class = (
-            SquashedGaussian if config["normalize_actions"]
-            else DiagGaussian)
+        action_dist_class = (SquashedGaussian
+                             if config["normalize_actions"] else DiagGaussian)
     return action_dist_class
 
 
-def get_log_likelihood(policy, model, actions, input_dict, obs_space,
-                       action_space, config):
-    model_out, _ = model({
-        "obs": input_dict[SampleBatch.CUR_OBS],
+def get_distribution_inputs_and_class(policy,
```

Comment:
`forward_fn` (get_distribution_inputs_and_class) done for SAC as well. Replaces both `action_sampler_fn` and `log_likelihood_fn`.